src/octoprint/plugins/classicwebcam/static/js/classicwebcam.js

```diff
@@ -0,0 +1,359 @@
+$(function () {
```

Comment:
Am I right in assuming that this is basically just what used to live in `src/static/js/app/viewmodels/control.js`?

Reply:
Yes, but with some changes to accommodate the new setup, e.g. webcam visibility is now managed globally and the new `onWebcamVisbilityChange` callback is used.